doc/en/getting-started.rst

```diff
@@ -9,7 +9,7 @@ Get Started
 Install ``pytest``
 ----------------------------------------
 
-``pytest`` requires: Python 3.6, 3.7, 3.8, 3.9, or PyPy3.
+``pytest`` requires: Python 3.7, 3.8, 3.9, or PyPy3.
```

Comment:
3.10 still seems to be missing from the list?

Reply:
Alternatively, how about:

```suggestion
``pytest`` requires: Python 3.7+ and PyPy3.
```

Reply:
Sorry, I forgot to fetch upstream changes and force pushed my local branch :man_facepalming: 
Fixed!